main.go

```diff
@@ -450,6 +451,20 @@ func init() {
 		}
 		log.Printf("[INFO]  : Enabled Outputs : %s\n", outputs.EnabledOutputs)
```

Comment:
```suggestion
		log.Printf("[INFO]  : Enabled Outputs : %s\n", outputs.EnabledOutputs)
```

My bad, this line should not be there